<launch>
  <group ns="robot_type1">
    <node pkg="2robots_server" type="server.py" name="server1" output="log" respawn="false">
        <param name="robot_type" value="type1"/>
    </node>
    <node pkg="2robots_server" type="client.py" name="client1" output="log" respawn="false">
        <param name="robot_type" value="type1"/>
    </node>
  </group>
  <group ns="robot_type2">
    <node pkg="2robots_server" type="server.py" name="server1" output="log" respawn="false">
        <param name="robot_type" value="type2"/>
    </node>
    <node pkg="2robots_server" type="client.py" name="client1" output="log" respawn="false">
        <param name="robot_type" value="type2"/>
    </node>
  </group>
</launch>